lib/python/cs/imageutils.py: cs.imageutils,iso14496: the classifiers have changed :-(
